### ISSUE TITLE
Bosses corpse only lootable by one

### ISSUE THREAD

**Issue by gianclaudio-giordano** [projectblitz/tracker#25]
In case of Group-loot, the boss of any instances are only lootable by the player in turn, rather than by everyone in the party. The roll frame though is working fine, it appears as soon as the one assigned to the loot right-clicks the boss.

**Comment by warlorddrak** [comment]
So the issue is that no one gets the roll window when the one player in turn has not looted it yet?

**Comment by gianclaudio-giordano** [comment]
Exactly, because the rool windows are not produced until first looting action. The players not in turn can't even see the actual drops, since the mobs are totally "un-rightclickable". I killed a Boss dualboxing with Group Loot mode:

Rotterz POV (he was in turn for looting according to the succession of loots): `https://imgur.com/a/uZRQDli`

My warlock could not interact with the boss: `https://imgur.com/a/BwxqTPo`

after I produced the loots from Rotterz POV, the rolls appeared in my screen tho, but the boss still uninteractable: `https://imgur.com/a/UxJGle4`

**Comment by gianclaudio-giordano** [comment]
Adding: In case the creature has a quest item to drop, then all the players interested (the ones who have the quest) can actually loot the corpse. Otherwise don't. Tested with [The Captain's Chest] quest in Booty Bay
